### PR TITLE
Fix linting errors

### PR DESCRIPTION
**Pull request by holgerd77**
This fixes the linting errors to get the build passing again. Tested by running the tests and looked if coverage is touching the effected lines.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ Blockchain.prototype.getHead = function (name, cb) {
     if (!hash) {
       return cb(new Error('No head found.'))
     }
-    self.getBlock(new Buffer(hash, 'hex'), cb)
+    self.getBlock(Buffer.from(hash, 'hex'), cb)
```
Comment (axic) [inline index.js]: For this you also need to include the `safe-buffer` package.

File: index.js [hunk 2]
```diff
@@ -308,13 +308,11 @@ Blockchain.prototype.getBlock = function (blockTag, cb) {
...
     lookupByHash(blockTag, cb)
-    return
   } else if (Number.isInteger(blockTag)) {
     async.waterfall([
       (cb) => lookupNumberToHash(blockTag, cb),
       (blockHash, cb) => lookupByHash(blockHash, cb)
     ], cb)
-    return
   } else {
     return cb(new Error('Unknown blockTag type'))
```
Comment (axic) [inline index.js]: If you remove `return` above, you can also remove it from here.

Comment (holgerd77) [reply index.js]: ? It's both removed. Or do you mean the ``return cb()``statement?

Comment (axic) [reply index.js]: Yep the return from this statement.

Comment (holgerd77) [reply index.js]: So remove the whole ``else`` branch? Sorry, can't see atm why this is safe to do, is it clear that this is never reached?

Comment (axic) [reply index.js]: I mean that if you remove the `return` statement you can omit it from all three branches (but keep the `cb`).

**Comment by holgerd77** [comment]
Done.

**Pull request merged**